configure/os/CONFIG.Common.RTEMS-mvme2700

```diff
@@ -1,7 +1,11 @@
 #
 # Author: Matt Rippa
 #
+ifeq ($(shell test $(RTEMS_VERSION) -gt 5; echo $$?),0)
+RTEMS_BSP = mvme2700
+else
 RTEMS_BSP = mvme2307
+endif
```

Comment:
I'm confused by this condition.  And for that matter, why ce876d6f114fa1e293c1b278bc11f05bba4cb886 changed mvme2700 to mvme2307 in the first place.  (@anjohnson do you remember what motivated this?)

Reply:
I'm not aware of anyone in the EPICS world using an actual MVME2307 board (Dirk might have some though); Gemini and APS both use the MVME2700 which was a later derivative with I believe similar or identical onboard I/O devices. Thus I wanted EPICS to use that model number in our target name. I'm guessing that RTEMS 6 now provides an mvme2700 BSP, but RTEMS 5 used the older model number in the BSP name.

Reply:
I recently added the `powerpc/mvme2700` BSP to RTEMS 6. I have no immediate plans to update the RTEMS 5 branch to have a `mvme2700`. If that was added would that help? I am not sure how far back compatibility with RTEMS versions EPCIS 7 extends?

If we plan a change to RTEMS 5 can this change be included until RTEMS is released with an `mvme2700` BSP? Note, this assumes it is easy to add the BSP to RTEMS 5. I have not looked into it.

Reply:
I'm not sure the an RTEMS change is warranted.  I think we (EPICS devs.) have to own the consequences of making the EPICS target `RTEMS-mvme2700` actually use the `mvme2307` RTEMS BSP.

Is there any benefit to trying to change this name mapping going forward?

Reply:
I've long since requested to properly identify the MVME2700 bsp by the physical board name 2700, not 2307. When Eric Norum helped me test the MVME2700 back around 2009, the change was made in EPICS and RTEMS 4.9. But when RTEMS 4.10 was released this reverted back to MVM2307.  

This is all just semantic, but sound bookkeeping would favor using the 2700 model number and updating EPICS configs to call that name in RTEMS too. I support your best judgement on the issue.

Reply:
If I make RTEMS 5 support the MVME2700 is that enough for EPICS? Does EPICS 7 support 4.10?

Reply:
EPiCS would still need to update ce876d6 to support the change in RTEMS.